ansible/roles/omero-python-deps/tasks/main.yml

```diff
@@ -1,12 +1,15 @@
 ---
 # tasks file for roles/omero-runtime
 
-- name: omero python deps | check variables
-  fail:
-    msg: "Invalid value of omero_python_deps_profile: {{ omero_python_deps_profile }}"
-  when: "omero_python_deps_profile not in ['distribution', 'pypi']"
 
-# TODO: Consider installing some of these with pip if the user requests it?
+- name: omero python deps | pypi pillow uninstall
```

Comment:
Can you avoid uninstalling/removing packages:
- If other packages depend on this it may break them (e.g. in a previous PR removing `python-pillow` also removed `cloud-init`)
- In this particular case the global `pip` will also remove the files installed by yum, but the yum/rpm database still lists the package as installed, e.g.

```
[root@5a85874b4943 /]# yum install -q -y epel-release

[root@5a85874b4943 /]# yum install -q -y python-pip

[root@5a85874b4943 /]# pip freeze |grep Pillow

[root@5a85874b4943 /]# yum install -q -y python-pillow

[root@5a85874b4943 /]# pip freeze |grep Pillow
Pillow==2.0.0

[root@5a85874b4943 /]# python -c 'import PIL'

[root@5a85874b4943 /]# pip uninstall -y Pillow
Uninstalling Pillow-2.0.0:
  Successfully uninstalled Pillow-2.0.0

[root@5a85874b4943 /]# pip freeze |grep Pillow

[root@5a85874b4943 /]# rpm -q python-pillow
python-pillow-2.0.0-19.gitd1c6db8.el7.x86_64

[root@5a85874b4943 /]# python -c 'import PIL'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named PIL
```